meetings/api/migrations/0001_initial.py

```diff
@@ -20,14 +20,17 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='Conference',
             fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
I don't think we need any migrations in this module. We don't have any models defined in the api module.